### ISSUE TITLE
[videojs 5.x] videojs.Button is undefined

### ISSUE THREAD

**Issue by smad2005** [SunnyLi/videojs-ass#4]
>TypeError: videojs.Button is undefined

```js
108L: AssButton = videojs.Button.extend();
```
i'm using instruction from https://github.com/videojs/video.js#quick-start

```html
 <link href="http://vjs.zencdn.net/5.0/video-js.min.css" rel="stylesheet">
 <script src="http://vjs.zencdn.net/5.0/video.min.js"></script>
```

**Comment by SunnyLi** [comment]
`videojs 5.0` made many internal API changes which is what caused this breakage since `videojs.Button` was part of the internal API that really shouldn't have been used..

For now the latest `videojs 4.x` version should still work and I will eventually update this plugin to support `videojs 5.x` in the near future.

**Comment by SunnyLi** [comment]
If losing the ability to toggle subtitle visibility is not a major concern (that subtitle will always be visible), then you can [disable the toggle button creation step](https://github.com/SunnyLi/videojs-ass/issues/2#issuecomment-71130958) so that this plugin will continue to work on `videojs 5.0`.

**Comment by smad2005** [comment]
Thank you

**Comment by anonym24** [comment]
video.js 5 can also response the player ( `data-setup='{"fluid":true} ` ), but if it's enabled subtitles aren't shown correctly.
Is it possible to fix it?

**Comment by SunnyLi** [comment]
Sorry for the late response. I'm working on updating the plugin to support `videojs 5.x` now (so that the button doesn't break the plugin initialization) and will see if I can also get responsive sizing to work. It'll probably be put in a `v5.x` branch since it will definitely break `videojs 4.x` support.

**Comment by SunnyLi** [comment]
I've released `v0.5` which adds support for `videojs 5.x` and have partial support for `fluid` mode.

**Comment by smad2005** [comment]
Thanks, my issue is resolved.
